Sources/NIO/Bootstrap.swift

```diff
@@ -185,6 +185,9 @@ public final class ServerBootstrap {
         return bind0(makeServerChannel: makeChannel) { (eventGroup, serverChannel) in
```

Comment:
Should `bind0` do this error handling? There are a few other paths through this bootstrap that didn't get changed here but that still do registrations.

Reply:
done now, thanks